lib/index.js

```diff
@@ -41,12 +41,17 @@ Server.prototype._createServer = function() {
     return;
   }
 
+  // connections
+  self._connections = [];
+
   var server = net.createServer(function(connection) { //'connection' listener
     new Connection(connection, self, {
       version: self.protocolVersion,
       authKey: self.authKey,
       protocol: self.protocol
     });
+    // push to connections array
+    self._connections.push(connection);
```

Comment:
any suggestions of the best way to set/get ID of connection?
do you think using stringified `connection.address()` would OK? (theoretically it should be unique)